hacl-rs/src/lowstar/endianness.rs

```diff
@@ -2,52 +2,64 @@ use core::convert::TryInto;
 
 // Little Endian
 
+#[inline]
 pub fn load16_le(bytes: &[u8]) -> u16 {
     u16::from_le_bytes(bytes[0..2].try_into().unwrap())
 }
 
+#[inline]
 pub fn store16_le(bytes: &mut [u8], x: u16) {
     bytes[0..2].copy_from_slice(&u16::to_le_bytes(x))
 }
 
+#[inline]
 pub fn load32_le(bytes: &[u8]) -> u32 {
     u32::from_le_bytes(bytes[0..4].try_into().unwrap())
 }
 
+#[inline]
 pub fn store32_le(bytes: &mut [u8], x: u32) {
     bytes[0..4].copy_from_slice(&u32::to_le_bytes(x))
 }
 
+#[inline]
```

Comment:
Did you check if this makes a difference for performance?

Reply:
I have not measured performance of this at all, no